mock/py/mockbuild/backend.py

```diff
@@ -105,10 +105,12 @@ def scrub(self, scrub_opts):
                 self.plugins.call_hooks('clean')
                 if self.bootstrap_buildroot is not None:
                     self.bootstrap_buildroot.plugins.call_hooks('clean')
+
                 for scrub in scrub_opts:
                     self.plugins.call_hooks('scrub', scrub)
                     if self.bootstrap_buildroot is not None:
                         self.bootstrap_buildroot.plugins.call_hooks('scrub', scrub)
+
```

Comment:
This whitespace change was intentional. Previously it was squashed so much together, that it was had to read